### PR TITLE
add web-show example

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve? 

add [web-show](https://github.com/chaos-mesh/web-show) examples for demo

### What is changed and how does it work?

use Deployment to deploy web-show 

### Check List 

 - Manual test (add detailed scripts or steps below)

### CODE REVIEW

File: examples/web-show/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+TARGET_IP=$(kubectl get pod -n kube-system -o wide| grep kube-controller | head -n 1 | awk '{print $6}')
+
+sed "s/TARGETIP/$TARGET_IP/g" deployment.yaml > deployment-target.yaml
+
+docker pull
```
Comment (yeya24) [inline examples/web-show/deploy.sh]: Do we need this?

File: examples/web-show/network-delay.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      - default
+    labelSelectors:
+      "app": "web-show"
+  delay:
+    latency: "10ms"
+    correlation: "100"
+    jitter: "0ms"
+  duration: "30s"
+  scheduler:
+    cron: "@every 60s"
```
Comment (yeya24) [inline examples/web-show/network-delay.yaml]: new line

File: examples/web-show/service.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  name: web-show
+  labels:
+    app: web-show
+spec:
+  selector:
+    app: web-show
+  ports:
+    - protocol: TCP
+      port: 8081
+      targetPort: 8081
```
Comment (yeya24) [inline examples/web-show/service.yaml]: ditto

File: examples/web-show/deploy.sh (hunk 1 above)
Comment (zhouqiang-cl) [reply examples/web-show/deploy.sh]: docker pull what?

Comment (cwen0) [reply examples/web-show/deploy.sh]: I have fixed it, PTAL again! thanks!

File: examples/web-show/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+kubectl apply -f service.yaml
+kubectl apply -f deployment-target.yaml
+
+rm -rf deployment-target.yaml
+
+while [[ $(kubectl get pods -l app=web-show -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; do echo "Waiting for pod running" && sleep 1; done
+
+kill $(lsof -t -i:8081) 2>&1 >/dev/null | True
+
+nohup kubectl port-forward svc/web-show 8081:8081 >/dev/null 2>&1 &
```
Comment (yeya24) [inline examples/web-show/deploy.sh]: Umm, actually I think just `kubectl port-forward svc/web-show 8081:8081` is fine. Is it necessary to make it run at background?

Comment (cwen0) [reply examples/web-show/deploy.sh]: I think if this script is blocked, it is easy for users to misunderstand that it was not successfully installed.

File: examples/web-show/deploy.sh [hunk 5]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+    cat << EOF
+This script is used to install wob-show.
```
Comment (Gallardot) [inline examples/web-show/deploy.sh]:
```suggestion
This script is used to install web-show.
```

File: examples/web-show/deploy.sh (hunk 4 above)
Comment (zhouqiang-cl) [reply examples/web-show/deploy.sh]: LGTM

**Comment by cwen0** [comment]
> A small nit otherwise looks good! But can I ask why add an empty `deployment-target.yaml`?

In `deployment-target.yaml`, the script will use a target IP instead of `TARGET` and I just want to keep the `deployment.yaml` without change

**Pull request merged**